rctv/main.py

```diff
@@ -30,6 +32,21 @@
 all_apps = json.loads(Path("rctv/apps.json").read_text())
 security = HTTPBasic()
 
+# This is 'bad' only doing this so we can load app-sdk.js from localhost
+origins = ["*"]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
```

Comment:
@gregsadetsky I added this for testing this locally. We need it eventually if we want people to host their own apps at `mydomain.com` and also use `<script src="rctv.recurse.com" />` because of CORS. But this is too permissive, so remind me to dial this back later.